modules/virt-configuring-default-cpu-model.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-default-cpu-model.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-default-cpu-model_{context}"]
+= Configuring the default CPU model
+
+Configure the `defaultCPUModel` by updating the `HyperConverged` custom resource (CR). You can change the `defaultCPUModel` while OpenShift Virtualization is running.
+
+Note: The `defaultCPUModel` is case sensitive.
+
+.Prerequisites
+
+* Install the OpenShift CLI (oc).
+
+.Procedure
+
+. Open the `HyperConverged` CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
+
+. Add the `defaultCPUModel` field to the CR and set the value to the name of a CPU model that exists in the cluster:
+
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+ name: kubevirt-hyperconverged
+ namespace: openshift-cnv
+spec:
+  defaultCPUModel: "EPYC"
+----
+
+. Apply the YAML file to your cluster.
```

Comment:
normally there is a command at this step to apply config files to the cluster, something like:
```suggestion
. Apply the YAML file to your cluster:
[source,terminal]
----
$ oc apply -f <config-name>.yaml
----
```
i'm not totally sure how it would look for your use case, but an SME would know for sure 🙂

Reply:
@iholder101 Please review the code sample here and let me know if it requires any update or is good to ad to the doc as-is.

Reply:
I don't think that's necessary as you already provide a command [here](https://github.com/openshift/openshift-docs/pull/60760/files#diff-9dd043aed0a1fd14f9c9175ba0d222618c45195515b39fb68a9c0827a3305912R26):

```
oc edit hco -n openshift-cnv kubevirt-hyperconverged
```

Reply:
Thanks @iholder101 !